CHANGELOG.md

```diff
@@ -12,7 +12,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Hypothesis strategies for acquisition functions
 - `Kernel` base class allowing to specify kernels
 - `MaternKernel` class can be chosen for GP surrogates
-- `hypothesis` strategies and roundtrip test for kernels, constraints and objectives
+- `GammaPriors` class can be chosen for `MaternKernel`s
+- `hypothesis` strategies and roundtrip test for kernels, constraints, objectives and priors
 
 ### Changed
```

Comment:
**NOTE** There seems to be an unrelated error in the doc building due to some stuff that seems to have changed in other packages. I am currently investigating that and will resolve here once this was fixed.

Reply:
Will be fixed by another PR